l10n_it_location_nuts/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,20 @@
+**Italiano**
+
+Dopo aver installato il modulo, aprire il wizard:
+
+Contatti / Configurazione / Localizzazione / Import NUTS 2013
```

Comment:
s/il wizard/la procedura guidata

Ho appena aggiunto la traduzione in "base_location_nuts", quindi:
s/Import/Importa

Reply:
non l'avevo vista, fatto!